application/src/uuidmanager.cpp

```diff
@@ -3,34 +3,51 @@
 #include <QJsonDocument>
 #include <QJsonObject>
 #include <QJsonParseError>
+#include <QStandardPaths>
 #include <QFile>
 #include <QDir>
 
-static const QString FILE_PATH = QDir::homePath() + "/UUID.txt";
 
-UUIDManager::UUIDManager(QObject *parent)
+static const QString FILE_PATH = QStandardPaths::writableLocation(QStandardPaths::ConfigLocation) + "/rheda-messenger";
+
+UuidManager::UuidManager(QObject *parent)
     : QObject{parent}
 {
 
 }
 
-bool UUIDManager::exists()
+bool UuidManager::exists()
 {
     return QFile::exists(FILE_PATH);
 }
 
-void UUIDManager::create(const QString &id)
+void UuidManager::create(const QString &id)
 {
     QByteArray textStream = id.toUtf8();
 
+    QDir dir;
+    dir.mkpath(FILE_PATH);
+
     QFile uuidFile(FILE_PATH);
     if (uuidFile.open(QIODevice::WriteOnly)) {
         uuidFile.write(textStream);
         uuidFile.close();
     }
 }
 
-void UUIDManager::clear()
+void UuidManager::clear()
 {
     QFile::remove(FILE_PATH);
 }
+
+QString UuidManager::getId() const
+{
+    QFile uuidFile(FILE_PATH);
+    QString idValue;
+    if (!uuidFile.open(QIODevice::ReadOnly | QIODevice::Text)){
```

Comment:
```suggestion
    if (!uuidFile.open(QIODevice::ReadOnly | QIODevice::Text)) {
```
code-style
Для справки https://wiki.qt.io/Qt_Coding_Style/ru